esrally/rally.py

```diff
@@ -898,6 +900,21 @@ def configure_reporting_params(args, cfg):
     cfg.add(config.Scope.applicationOverride, "reporting", "numbers.align", args.report_numbers_align)
 
 
+def ensure_internet_connection(args, cfg, logger):
```

Comment:
For what it's worth, I'm not really happy with the name, there are two cases where no Internet connection is needed at all, but I haven't found a better name! :shrug: